x-pack/plugins/enterprise_search/public/applications/app_search/components/schema/components/schema_callouts.test.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { setMockValues, setMockActions } from '../../../../__mocks__';
+import '../../../__mocks__/engine_logic.mock';
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { SchemaErrorsCallout } from '../../../../shared/schema';
+
+import {
+  UnsearchedFieldsCallout,
+  UnconfirmedFieldsCallout,
+  ConfirmSchemaButton,
+} from './schema_callouts';
+
+import { SchemaCallouts } from './';
+
+describe('SchemaCallouts', () => {
+  const values = {
+    hasUnconfirmedFields: false,
+    hasNewUnsearchedFields: false,
+    mostRecentIndexJob: {
+      hasErrors: false,
+      activeReindexJobId: 'some-id',
+    },
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    setMockValues(values);
+  });
+
+  it('renders nothing if there is nothing to call out', () => {
+    const wrapper = shallow(<SchemaCallouts />);
+
+    expect(wrapper.text()).toBeFalsy();
```

Comment:
FYI that `isEmptyRender()` doesn't work here because it technically renders as a `<Fragment />` (even tho that fragment is empty). I found `.text()` to be the quickest way to functionally check for a falsy value.